pkg/apis/controlplane/types.go

```diff
@@ -213,6 +213,8 @@ type NetworkPolicyRule struct {
 	// action “nil” defaults to Allow action, which would be the case for rules created for
 	// K8s NetworkPolicy.
 	Action *secv1alpha1.RuleAction
+	// RuleName user can store extra information to describe this rule
+	RuleName string
```

Comment:
i think we can skip adding rule name here until we decide to implement some sort of usage of this field in agents

Reply:
Sure thing